packages/react-reconciler/src/ReactFiberContext.js

```diff
@@ -137,7 +137,7 @@ function pushTopLevelContextObject(
   didChange: boolean,
 ): void {
   invariant(
-    contextStackCursor.cursor == null,
+    contextStackCursor.current == null,
```

Comment:
This looks like it caught a bug! The invariant could never fire because of the typo.

Reply:
Actually it fires in tests. 😮

Reply:
Haha, nice 😅